compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/model/SymbolTable.java

```diff
@@ -882,11 +882,11 @@ private void defineNilableIntegerBitwiseOperations(OperatorKind opKind) {
         for (int i = 0; i < unsignedNilableIntTypes.length; i++) {
```

Comment:
Should the method name be `defineNilableIntegerBitwiseOrAndXorOperations`?

Reply:
Yeah will change it